crates/floresta-chain/src/lib.rs

```diff
@@ -21,6 +21,9 @@ pub use pruned_utreexo::error::*;
 pub use pruned_utreexo::udata::*;
 pub use pruned_utreexo::Notification;
 
+/// Simple enum representing the network type.
+///
+/// Possible values are Bitcoin, Testnet, Regtest, and Signet.
 #[derive(Debug, Clone, Copy, PartialEq, Eq)]
```

Comment:
As per the [contributing.md](https://github.com/vinteumorg/Floresta/blob/master/CONTRIBUTING.md#coding-conventions):

> If you need an attribute, use the attribute before the docstring

Please swap the attributes (this #[derive]) with the comment

Reply:
We still have a lot to do before we can fully conform with these guidelines. My current approach is to change stuff as we go, not create PRs specifically for that. Refactor PRs are usually a PITA for other contributors because they tend to be a conflict bomb.

I think it's better to just change where you're touching, and dealing with the others as we go.